addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -78,6 +78,7 @@ $o-we-toolbar-color-clickable-active: $o-we-bg-darkest;
 
     .o_we_colorpicker_switch_pane_btn {
         flex: 0 0 auto;
+        width: 60px;
```

Comment:
This seems really strange but maybe, so I went and test it. Here is what I found:
1. I can't reproduce the original bug without your branch. Did you succeed ? I tested in both Chrome and Firefox. In both browsers I get something like this:
![image](https://user-images.githubusercontent.com/10338094/161944276-393bcb32-1ee9-4cc5-92be-413e1501d674.png)

So, not pretty but not as bad as in the ticket (it stays in the panel).

2. I tested with your fix and I get this result:
![image](https://user-images.githubusercontent.com/10338094/161944769-097bfe2b-231b-4871-9038-2a9b85d5010b.png)
... arguably worse in that language, layout is better but the buttons are not readable anymore... although there is still space to show some more letter. So even if the original bug is possible to reproduce, I don't think this is the right solution, I know CSS and flex is annoying but there has to be a way to make the buttons fill the space and make the ellipsis work properly (btw you see there is an ellipsis after the trash icon... that's probably to solve at the same time)
3. The CSS rule on its own is very strange, if it make sense it lacks comments in the code. But what I see here is "those buttons cannot shrink or grow and we fix them to 60px"... seems bad. But actually there is another rule for when the colorpicker is in the editor panel here: https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L1747 -> and that allows the buttons to shrink and grow. Maybe you should modify something over there instead, I don't think there is an issue when the colorpicker is outside the panel, right? :thinking: 

Now for the commit:
- First, I wanted to have a look at the ticket to find more information (I only remembered the problem from what we saw yesterday)... so I used the OPW you mention in the commit message: this is not the right one :/ See, why we insisted on this yesterday...? :wink: The OPW you mention is a bug of background color for the searchbar (which we also discussed yesterday).
- Other convention we want to respect in the commit message:
    - the title must complete the sentence "When applied, this commit will ...". Here something like `[FIX] web_editor: properly display colorpicker buttons in all languages`
    - the title should not start with an uppercase (the inverse of the git most common convention, but it's the Odoo one :shrug: )
    - Both title and message should not exceed 72 characters per line (and you should use as much as those 72 characters of course)

At last, in whatever way we fix this: should not this target 14.0 ? We always try to fix bugs where they were created, especially when the fix is probably the same on both places :+1:

Reply:
Yes, I was able to reproduce it. This is the ticket opw-2786778 I used the wrong branch to put my fix so my script messed up with the ticket id :/ I'll look for a better fix anyway